README.md

```diff
@@ -27,25 +27,23 @@ Alternatively in Swift:
 
 ```swift
 centralManager = RZBCentralManager()
-
-centralManager!.scanForPeripheralsWithServices([CBUUID.rzb_UUIDForHeartRateService()], options: nil, onDiscoveredPeripheral: { (peripheral: CBPeripheral?, advInfo: [NSObject : AnyObject]?, RSSI: NSNumber?) in
-    guard let centralManager = self.centralManager, peripheral = peripheral else { return }
-    centralManager.stopScan()
-    self.peripheral = peripheral
-    peripheral.rzb_addHeartRateObserver({ (measurement: RZBHeartRateMeasurement?, error: NSError?) in
+centralManager.scanForPeripheralsWithServices([CBUUID.rzb_UUIDForHeartRateService()], options: nil) { scanInfo, error in
+    guard let peripheral = scanInfo?.peripheral else {
+        print("ERROR: \(error!)")
+        return
+    }
+    self.centralManager.stopScan()
+    peripheral.addHeartRateObserver({ measurement, error in
         guard let heartRate = measurement?.heartRate else { return }
         print("HEART RATE: \(heartRate)")
-    }, completion: { (error: NSError?) in
+    }, completion: { error in
         guard let error = error else { return }
         print("ERROR: \(error)")
     })
-}, onError: { (error: NSError?) in
-    guard let error = error else { return }
-    print("ERROR: \(error)")
-})
+}
  ```
 
-This block will wait for bluetooth to power on and scan for a new peripheral supporting the heart rate service. When one is found, the app will connect to the peripheral, discover the heart rate service and observe the characteristic. When the characteristic is notified, the `NSData*` object is serialized into a more developer friendly object. All of these details are nicely encapsulated for you, and the pattern of CBPeripheral categories should be easily extendable to your devices domain space.
+This block will wait for bluetooth to power on and scan for a new peripheral supporting the heart rate service. When one is found, the app will connect to the peripheral, discover the heart rate service and observe the characteristic. When the characteristic is notified, the `NSData*` object is serialized into a more developer friendly object. All of these details are nicely encapsulated for you, and the pattern of extending `RZBPeripheral` via categories should easily extend to your devices domain space.
```

Comment:
Readme is pretty solid at explaining everything except "extending `RZBPeripheral` via categories should easily extend to your devices domain space" is a little confusing. Might just want to take it out or explain it later on in the doc